spec/develop.js

```diff
@@ -56,13 +56,15 @@ describe('development server', function () {
     })
     .then(function (body) {
       assert(body.length);
+      assert(body.indexOf('<!doctype html>') === -1);
```

Comment:
Yeah, nice idea - though I'd like to postpone that, as it's not really about hops, but rather webpack-dev-server behaviour.